app/controllers/application_controller.rb

```diff
@@ -7,21 +7,28 @@ class ApplicationController < ActionController::Base
   # Before Actions #############################################################
 
   def user_has_position(position)
-    unless current_user && current_user.position == position
-      flash[:notice] = "Sorry but you must be the #{position} to view that page"
+    unless current_user && !current_user.position.nil? &&
+           current_user.position == position
```

Comment:
If one person has two positions, it will only use the first position they were assigned. If one position is assigned to several users this will allow both users through.